phpunit.xml.dist

```diff
@@ -10,7 +10,7 @@
     </testsuites>
 
     <filter>
-        <whitelist>
+        <whitelist processUncoveredFilesFromWhitelist="true">
```

Comment:
We should also create a template for `phpunit.xml` in the [maintainers repository](https://github.com/zendframework/maintainers).

Reply:
Good point 👍 I'll add it soon.